jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnection.java

```diff
@@ -885,8 +886,10 @@ private void releaseChunk()
         @Override
         protected void onCompleteSuccess()
         {
+            boolean upgrading = _info != null && _info.getStatus() == HttpStatus.SWITCHING_PROTOCOLS_101;
```

Comment:
I'm not sure this is enough now.  Don't we have to also deal with the 200 OK for `CONNECT` requests?   See below:

Reply:
That wasn't really in the scope of this PR, @sbordet said he was going to build on this in a subsequent PR to get the `CONNECT` working.

Reply:
Perhaps, but the change I suggest below should also work for the purposes of this PR and at least has a chance of being right for a subsequent `CONNECT` PR.    I'm not saying add tests for `CONNECT` in this PR, as you can leave that to @sbordet's subsequent PR.